docs/changelog.rst

```diff
@@ -15,6 +15,7 @@ Changelog
 * :bug:`-` Fix an edge-case problem with synchronizing database with the server.
 * :bug:`-` Users will be able to edit EVM tokens again and the information about underlying tokens will be correctly displayed.
 * :bug:`-` Fix an issue with nginx websocket proxying on docker.
+* :bug:`-` Now `Card Cashback Reversal` entries from cryptocom csv will be imported correctly.
```

Comment:
Can be changed in other PR since all the tests are green

```suggestion
* :bug:`-` Now `Card Cashback Reversal` entries from cryptocom csv will be correctly imported.
```